src/serializer/AlgoliaSerializer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace rias\scout\serializer;
+
+class AlgoliaSerializer extends \League\Fractal\Serializer\ArraySerializer
+{
+    /**
+     * Serialize a collection.
+     *
+     * @param string $resourceKey
+     * @param array  $data
+     *
+     * @return array
+     */
+    public function collection($resourceKey, array $data)
+    {
+        if ($resourceKey === false) {
+            return [$resourceKey => $data];
```

Comment:
Shouldn't this be reversed? If the resourceKey is false then just return the data?